### PR TITLE
Add Keygen

### PR DESCRIPTION
**Pull request by ezekg**
Happy Monday! Stoked about this initiative. Blog post: https://keygen.sh/blog/keygen-joins-osspledge/

Let me know if there's anything you'd like to see changed.

### Assets

- Logo: https://assets.keygen.sh/logos/logo.svg
- Logomark: https://assets.keygen.sh/logos/logomark.svg

### CODE REVIEW

**Review by vladh — APPROVED** [review]

**Comment by vladh** [comment]
Great to have you on board, and I love your post! 🎉 

LGTM, but waiting for a second review.

**Comment by chadwhitacre** [comment]
Woo-hoo! I think you're the first one to include `monetaryValueOfMaterials`! Yay! Do you expand on this in the blog post? It's not jumping out at me. I think that would be really interesting to unpack to help other companies start thinking about this category.

**Pull request merged**
